src/main/kotlin/widgets/DataSelectionInterface.kt

```diff
@@ -0,0 +1,12 @@
+package widgets
+
+/**
+ * This interface is used in Selector as the data from which is selected in the widget
+ */
+
+interface DataSelectionInterface {
```

Comment:
Poor name; how about something like "ComboBoxSelectionItem" or something

Reply:
I'll go with that